mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -47,6 +47,13 @@ struct Chipset {
   DEFINE_COMP_OPERATOR(>)
   DEFINE_COMP_OPERATOR(>=)
 #undef DEFINE_COMP_OPERATOR
+
+  bool isGfx940() const {
+    return majorVersion == 9 && minorVersion >= 4 && minorVersion < 5;
+  }
+  bool hasOcpFp8() const {
+    return (majorVersion == 9 && minorVersion >= 5) || majorVersion >= 12;
+  }
```

Comment:
I don't think these belong in the chipset class, they can be defined as free functions where needed.